src/spring/azext_spring/_params.py

```diff
@@ -1291,3 +1291,12 @@ def prepare_common_logs_argument(c):
         c.argument('service', service_name_type)
         c.argument('execution', help='the name of the job execution.')
         c.argument('job', help='The name of job running in the specified Azure Spring Apps instance.')
+
+    with self.argument_context('spring private-dns-zone add') as c:
+        c.argument('service', service_name_type)
+        c.argument('zone_id', type=str, help='The resource id of the private DNS zone which you would like to configure with the service instance.')
```

Comment:
It is not necessary to specify `type=str`, since the param is string by default if not specify that.
```suggestion
        c.argument('zone_id', help='The resource id of the private DNS zone which you would like to configure with the service instance.')
```

Reply:
ok, removed.